src/katsdpcontroller/generator.py

```diff
@@ -831,10 +834,11 @@ def _make_fgpu(
                 "rx-missing-unixtime",
             ]:
                 fgpu.sensor_renames[f"input{j}-{name}"] = f"{stream.name}-{label}-{name}"
-        # Prepare expected data rate
+        # Prepare expected data rates etc
         fgpu.static_gauges["fgpu_expected_input_heaps_per_second"] = sum(
             src.adc_sample_rate / src.samples_per_heap for src in srcs
         )
+        fgpu.static_gauges["fgpu_expected_engines"] = 1.0
```

Comment:
I'm ... not sure why just 1? Ditto for the xbgpu case down below.

Reply:
`fgpu` is a single engine. The number of expected engines per engine is 1. The code that processes static_gauges takes a sum across all the engines.

Reply:
Ah. That explains things.